ksql-common/src/test/java/io/confluent/ksql/schema/ksql/ColumnTest.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.schema.ksql;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import com.google.common.testing.EqualsTester;
+import com.google.common.testing.NullPointerTester;
+import io.confluent.ksql.schema.ksql.types.SqlType;
+import io.confluent.ksql.schema.ksql.types.SqlTypes;
+import java.util.Optional;
+import org.junit.Test;
+
+public class ColumnTest {
+
+  @Test
+  public void shouldThrowNPE() {
+    new NullPointerTester()
+        .setDefault(SqlType.class, SqlTypes.BIGINT)
+        .setDefault(String.class, "field0")
+        .testAllPublicStaticMethods(Column.class);
+  }
+
+  @Test
+  public void shouldImplementEqualsProperly() {
+    new EqualsTester()
+        .addEqualityGroup(
+            Column.of("someName", SqlTypes.INTEGER),
+            Column.of("someName", SqlTypes.INTEGER)
+        )
+        .addEqualityGroup(
+            Column.of("someName".toUpperCase(), SqlTypes.INTEGER)
+        )
+        .addEqualityGroup(
+            Column.of("different", SqlTypes.INTEGER)
```

Comment:
nit: superfluous equality group. 

The group above tests case-sensitivity. But it its case sensitive on the name, it's definitely sensitive to different names.